pkgs/development/tools/misc/lint-staged/default.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
```

```suggestion
{ lib,
  buildNpmPackage,
  fetchFromGitHub,
}:

```

Are you using some code formatter?

---

Yeah, `alejandra`, my mistake. I thought `nixpkgs-fmt` would be preferred here but that doesn't seem to quite match your suggestion either so I've applied the fixes manually.

---

This is a bit disputed. There is no official formatter or format style yet.

Indeed there is the [RFC 101](https://github.com/NixOS/rfcs/pulls/101) that suggests nixpkgs-fmt, but it didn't reach final status yet.